src/user-guide/writing-workflows/scheduler.rst

```diff
@@ -0,0 +1,226 @@
+Scheduler Configuration
+=======================
+
+The :cylc:conf:`flow.cylc[scheduler]` section configures certain aspects of
+:term:`scheduler` behaviour at the workflow level.
+
+Many of these configurations can also be defined at the site-user level in
```

Comment:
Is `site-user` a formal term which we've defined somewhere?

If not I suggest that `user or site` would be more helpful to the reader.